src/main/java/com/easypost/model/Refund.java

```diff
@@ -38,7 +39,8 @@
         Map<String, Object> wrappedParams = new HashMap<String, Object>();
         wrappedParams.put("refund", params);
 
-        return request(RequestMethod.POST, classURL(Refund.class), wrappedParams, List.class, apiKey);
+        Refund[] response = request(RequestMethod.POST, classURL(Refund.class), wrappedParams, Refund[].class, apiKey);
```

Comment:
This isn't correct. The response of a Refund should be a refreshed Shipment object, not an array of refunds to list. The `refund_status` would eventually say `refunded`.

Reply:
This is **create()** method in the Refund class, I think you are talking about **refund()** method in the Shipment class.

Reply:
Ah, another janky endpoint. You are correct... it returns a list of JSON objects. This is fine then.